platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -116,6 +125,11 @@ public void add(String type, Map<String, Object> properties) throws PersistenceE
                     e);
         }
     }
+    @Override
+    // Input Map is expected to have the suffixes on the key names
```

Comment:
✏️  Since this is now mentioned in the interface's JavaDoc, we could remove this comment.